source/type/data.cpp

```diff
@@ -426,16 +426,30 @@ void Data::build_associated_calendar() {
     }
 }
 
+
+static void build_frames(navitia::type::VehicleJourney* vj){
```

Comment:
can you add a comment there to explain that this method builds all the frames' back references ?